test/specs/commonTests/implementsCommonProps.js

```diff
@@ -276,3 +276,36 @@ export const implementsWidthProp = (Component, widths = SUI.WIDTHS, options = {}
     }
   })
 }
+
+/**
+ * Assert that, when FormField Component receives an Id, the respective htmlFor prop
+ * is added to Label Component
+ * @param {React.Component|Function} Component The component to test.
+ * @param {String} propKey A props key.
+ * @param {Object} [options={}]
+ * @param {Object} [options.requiredProps={}] Props required to render the component.
+ */
+export const labelImplementsHtmlForProp = (Component, propKey, options = {}) => {
+  const { assertRequired } = helpers('labelImplementsHtmlForProp', Component)
+  const { requiredProps = {} } = options
+  describe(`${propKey} (common)`, () => {
+    assertRequired(Component, 'a `Component`')
+    it(`adds ${requiredProps.control} Id to Label's HtmlFor prop`, () => {
```

Comment:
We'll need generalize this test a bit more.  It shouldn't know which component it is testing.  We should remove the `control` awareness.

We will then be able to use it in the other `Form*` component tests:

```jsx
labelImplementsHtmlForProp(FormFieldButton)     // FormFieldButton-test.js
labelImplementsHtmlForProp(FormFieldCheckbox)   // FormFieldCheckbox-test.js
labelImplementsHtmlForProp(FormFieldDropdown)   // FormFieldDropdown-test.js
labelImplementsHtmlForProp(FormFieldGroup)      // FormFieldGroup-test.js
labelImplementsHtmlForProp(FormFieldInput)      // FormFieldInput-test.js
labelImplementsHtmlForProp(FormFieldRadio)      // FormFieldRadio-test.js
labelImplementsHtmlForProp(FormFieldSelect)     // FormFieldSelect-test.js
labelImplementsHtmlForProp(FormFieldTextarea)   // FormFieldTextarea-test.js
```

Suggest we don't take any arguments as this test will require the `id` of the top level component to present no the `label` subcomponent after it is rendered.